code/game/objects/items/granters.dm

```diff
@@ -430,3 +430,23 @@
 	icon_state = "cooking_learing_sweets"
 	oneuse = FALSE
 	remarks = list("So that is how icing is made!", "Placing fruit on top? How simple...", "Huh layering cake seems harder then this...", "This book smells like candy", "A clown must have made this page, or they forgot to spell check it before printing...", "Wait, a way to cook slime to be safe?")
+
+/obj/item/book/granter/crafting_recipe/pipegun_prime
+	name = "diary of a dead assistant"
+	desc = "A battered journal. Looks like he had a pretty rough life."
+	crafting_recipe_types = list(
+		/datum/crafting_recipe/pipegun_prime
+	)
+	icon_state = "book1"
+	oneuse = TRUE
+	remarks = list("He apparenty mastered some lost guncraftng technique.", "Why do I have to go through so many hoops to get this shitty gun?", "That much Grey Bull cannot be healthy...", "Did he drop this into a moisture trap? Yuck.", "Toolboxing techniques, huh? I kinda just want to know how to make the gun.", "What the hell does he mean by 'ancient warrior tradition'?")
+
+/obj/item/book/granter/crafting_recipe/pipegun_prime/turn_page(mob/user)
+	. = ..()
+	if(user.mind.assigned_role != "Assistant")
```

Comment:
I don't like job checks

Reply:
make it into a trait added by every other jobs skillchip or something

Reply:
Sure. I don't think assistants _have_ a skillchip but nothing stops them getting one. So maybe I'll add a skillchip for linguistics or something in the library to help decipher the text.

Reply:
I don't think the assistant job should be rewarded with anything.

Reply:
Maybe it's best to kill this check altogether.

Reply:
> I don't think the assistant job should be rewarded with anything.
![image](https://user-images.githubusercontent.com/53862927/105563146-ef45f300-5d14-11eb-8c34-26ae82b3ffc9.png)
This statement could be used in many places and it will be correct in all of them.

Reply:
I'm gonna just allow the absolute obscurity of oddity maint loot be the limitation for this one. It's quite, quite unlikely to turn up in a given round, and someone may not appreciate it for what it is by just seeing a book on a maint rack.